docs/variant-specific/pink.mdx

```diff
@@ -291,37 +291,33 @@ These conventions apply to any variant with a pink (touched by all ranks) suit.
   - Bob knows that Alice is violating _Pink Promise_.
-  - If he plays slot 1, then Alice will mark it as the matching 2 and will misplay it.
-  - Bob concludes this is a _Bubblegum Finesse_, with pink 1 in slot 2 and pink 2 in slot 1.
+  - If Bob blind-plays his _Finesse Position_ card and it successfully plays, it is guaraunteed to be one of the 1s. If that happened, then Cathy would mark her clued card as the matching 2 and will go on to misplay it.
+  - Thus, Bob concludes this is a _Bubblegum Finesse_, with pink 1 in slot 2 and pink 2 in slot 1.
   - Bob blind-plays his slot 2, but it is a red 1. It is now revealed to Bob that this was a _Bubblegum Bluff_ instead of a _Bubblegum Finesse_.
-  - Cathy knows that if her clued card were really a 2, Bob would have played slot 1. Thus, it cannot be a 2. Furthermore, there are no _Ejections_ that match this situation, so she must be holding a pink 3.
-  - No one is promised the pink 1 or 2, and the team is synchronized on the identity of Cathy's pink 3.
-- _Historical note:_ This was previously named a _Taffy Bluff_.
+  - Cathy knows that if her clued card were really a 2, Bob would have played slot 1. Thus, it cannot be a 2. Furthermore, there are no _Ejections_ that match this situation, so a _Bubblegum_ move must have happened. Normally, a _Bubblegum Bluff_ would be on a _one-away-from-playable_ card, but that would be the pink 2, and due to the circumstances of the clue, it is impossible to be a 2. Thus, it instead must be the _two-away-from-playable_ card, which is the pink 3.
 
 ### The Bubblegum Double Bluff
 
 - Normally, after a _Bubblegum Bluff_, the entire team is synchronized on the identity of the focus of the _Bubblegum_ move.
-- But what if the note the clue receiver would write on their card is still incorrect?
-- The clue-giver must be expecting additional blind-plays to resynchronize the information.
+- But what if the note that the clue receiver would write on their card is still incorrect? In this situation, the clue-giver expects additional blind-plays to resynchronize the information, in the same way that a normal _Double Bluff_ works.
 - For example, in a 4-player game:
   - It is the first turn and nothing is played on the stacks.
   - Alice clues number 3 to Donald, touching a pink 4 on slot 1.
-  - Bob knows that Alice is violating _Pink Promise_. This must mean that he is supposed to play his slot 3 card as pink 1, with pink 2 and 3 promised as playable through his current finesse position.
-  - Bob blind-plays slot 3, and it is a red 1.
-  - From Donald's perspective, he knows that Alice performed a _Bubblegum Bluff_. Donald marks his slot 1 card as a pink 2 (_one-away-from-playable_).
+  - Bob knows that Alice is violating _Pink Promise_. This must mean that he is supposed to play his slot 3 card as pink 1, with pink 2 and 3 promised as playable through his current _Finesse Position_ and _Second Finesse Position_, respectively.
```

Comment:
(d'oh, I just realized you have made edits to the PR...)

I think my original intention was to say that this is a finesse, hence the word "through". it could be layered, etc. 

not sure if the new language about "Second Finesse Position" and "respectively" is allowing for that? it's probably "clear enough" ? but maybe worth being extra clear by not mentioning it.

Reply:
> it could be layered

yeah but all finesses can be layered, so I don't think we need to specifically mention that.

Reply:
My point was that the language "through his _Finesse Position_" should be sufficient, without mentioning second finesse, since that may not even be where the card is located. I.e. the current language is more specific and hence inaccurate in some cases.